scripts/install.sh

```diff
@@ -41,9 +41,11 @@ VERSION=""
 CONTINUE=false
 FILE_STORAGE=""
 CONFIG_DIRECTORY=""
-SYSTEMD_DISABLED=""
 SYSTEMD_CONFIG=""
 
+# set by check_dependencies therefore cannot be set by set_defaults
+SYSTEMD_DISABLED=false
```

Comment:
Is any reason to move this variable to new line?

Reply:
visual separation